javascript/src/seedsource/containers/ReportButton.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux'
+import GroupedButton from 'seedsource/components/GroupedButton'
+import { createReport } from "../../actions/report";
+import PropTypes from 'prop-types'
+
+const mapStateToProps = (state, { name }) => {
+    return {
+        active: name === state.runConfiguration.unit
+    }
+}
+
+const mapDispatchToProps = (dispatch, { name }) => {
+    return {
+        onClick: () => {
+            dispatch(createReport(name))
+        }
+    }
+}
+
+GroupedButton.propTypes = {
+    name: PropTypes.string.isRequired
+}
+
+export default connect(mapStateToProps, mapDispatchToProps)(GroupedButton)
```

Comment:
Why does this use `GrouppedButton`? It's used in isolation so it's not actually grouped... Also, it's active if `name === state.runConfiguration.unit`? Does this component even need to exist? Why not just use `<button className="button" ...>`?